drivers/npmx/shell/adc.c

```diff
@@ -321,14 +321,4 @@ SHELL_STATIC_SUBCMD_SET_CREATE(sub_adc_ntc,
 SHELL_STATIC_SUBCMD_SET_CREATE(sub_adc, SHELL_CMD(meas, &sub_adc_meas, "ADC measurement", NULL),
 			       SHELL_CMD(ntc, &sub_adc_ntc, "ADC NTC", NULL), SHELL_SUBCMD_SET_END);
 
-void dynamic_cmd_adc(size_t index, struct shell_static_entry *entry)
```

Comment:
Creating dynamic commands can be squashed with this, to reduce unnecessary commit history.

Reply:
It is not possible to mix `SHELL_STATIC_SUBCMD_SET_CREATE` from `npmx_shell.c` file with `SHELL_SUBCMD_ADD`. It can compile, but it is not working as expected. Creating dynamic commands is a transition state not to create any regression between commits.